src/content/docs/tutorial-errors/scope-the-problem.mdx

```diff
@@ -0,0 +1,159 @@
+    to="https://one.newrelic.com/nr1-core?state=20fda75b-58fb-a92a-f9e1-7b052035c6e8"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+
+</TechTileGrid>
+
+Once you've installed an agent, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core?filters=(domain%3D'APM'ANDtype%3D'APPLICATION'))** and select your app. If you don't see much data just yet, step away for a while and let the agent gather real-time data as your application runs.
+
+* [Integrate your logs](/docs/logs/get-started/get-started-log-management/#integrate-logs). The APM agent provides some logging details with logs in context, but certain key attributes could be missing from your log details. We recommend that you invest some time in setting up our full logs solution to expedite the error analysis process.
+
+## Find the critical error [#critical-error]
+
+With your apps instrumented, New Relic can capture data about your services. This includes data about error occurrences in your app. 
+
+<Steps>
+    <Step>
+        ### Think about the end user
+
+Similar to our flood metaphor, it's likely that when an outage occurs, the alerting services only indicate a problem exists somewhere. When you first receive an alert, you won't have enough context to make a decision about where to start in your system. 
+
+For example, imagine two services that have begun alerting, one for logging in and one for searching inventory. You're only receiving reports that the search functionality is failing for end users, but the login function works fine. You know in this case that, while logging in is critical to your funnel, you should prioritize the service that directly affects your end user. You may believe that the log in function is more important, but it's critical to separate your beliefs from your observability practices. 
+
+This practice applies even if the end user hasn't reported anything. When you notice services failing, you can ask yourself these questions:
+
+* Is the end user experiencing a problem? 
+* How should their experience look?
+* What behavior are they currently experiencing?
+
+    </Step>
+    <Step>
+        ### Choose a service
+
+Let's see how this might look in practice. When you view the **All entities** page, you notice four services are alerting.
+
+<img
+title="Overview errors affecting your services"
+alt="A screenshot showing an app with many errors"
+src={allEntitiesView}
+/>
+
+After asking yourself the questions from step one, you know that: 
+
+* The end user is struggling with purchase actions.
+* Your site should only display in-stock items.
+* Your site is displaying all products, so customers are able to purchase out-of-stock items.
+
+Having worked out the nature of the problem, you know that `api-gateway` is a dependency for other services that keep track of your inventory. This is the start of your observability journey. 
```

Comment:
This sentence could get clearer. Something like, "You've performed an audit that tells you what's failing and it's x. Now you need to explore x in all entities to learn more." 

Obviously you will write this much better than me lol.